platform/broadcom/sonic-platform-modules-accton/as7326-56x/utils/accton_handle_idt.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+if [ -s /usr/local/bin/done_idt_init ];then
+    echo "There is a done_idt_init file"
+else
+    
+    cat /etc/init.d/opennsl-modules-3.16.0-5-amd64|grep mac_handle.sh
```

Comment:
>nnsl-modules-3.16.0-5-amd64|grep  [](start = 23, length = 33)

not correct kernel. fragile code.

Reply:
I has fixed incorrect kernel version to new commit at 97b44d1.